webpack.config.ts

```diff
@@ -3,6 +3,7 @@ import LiveReloadPlugin from 'webpack-livereload-plugin';
 import { merge } from 'webpack-merge';
 
 import grafanaConfig from './.config/webpack/webpack.config';
+// const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
We keeping these? I wouldn't mind, just checking.

Reply:
I deleted bundle analyzer before because I thought we were switching to the plugins-platform bundle analyzer github action. It seems like you use this locally though @gtk-grafana to monitor our package sizes? I'm fine leaving it in, apologies.